test/verify/check-storage-scaling

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
```

Comment:
We are in 2016 by now.. or something :-)

I don't have a strong opinion about this being an entirely new file, but IMHO it could just  be appended to check-storage-basic `testBasic`. That way it doesn't need the overhead of an entirely new VM boot, and there are already some other predictable drives visible.

Reply:
> I don't have a strong opinion about this being an entirely new file, but IMHO it could just be appended to check-storage-basic testBasic

We don't really want to have Cockpit open when the 200 block devices appear, so I think a separate test is best.

Reply:
Ah, I see. It could still log out, modprobe, and log back in, with a comment that doing this "online" is both too slow and also rather unrealistic. That's still faster than creating a new VM. But not such a big deal either way.